tests/Report/SummaryReportTest.php

```diff
@@ -30,7 +30,11 @@ public function getExamples() {
     $es[] = ['ex1.json', '4.7.31', 'upgrade', 'notice', 'CiviCRM Upgrade Available'];
     $es[] = ['ex1.json', '4.7.29', 'upgrade', 'notice', 'CiviCRM Upgrade Available'];
     $es[] = ['ex1.json', '4.6.36', 'upgrade', 'notice', 'CiviCRM Upgrade Available'];
-    $es[] = ['ex1.json', '4.5.10', 'upgrade', 'warning', 'CiviCRM Version End-of-Life'];
+    $es[] = ['ex1.json', '4.5.10', 'upgrade', 'critical', 'CiviCRM Security Upgrade Needed'];
+
+    $es[] = ['ex3.json', '5.9.1', 'upgrade', 'critical', 'CiviCRM Security Upgrade Needed'];
+    $es[] = ['ex3.json', '5.10.4', 'upgrade', 'warning', 'CiviCRM Version End-of-Life'];
```

Comment:
I know this has been discussed before but we shouldn't be calling 5.x versions end of life

Reply:
That does appear to be the policy:
https://github.com/civicrm/civicrm-pingback/blob/4323f4e68cfdcb495b71fb51f9027dda400315d9/versions.json#L624-L625

See #4 and [infra/ops#871](https://lab.civicrm.org/infra/ops/issues/871) for discussion of how to treat old versions.

The main thing is that regardless of what the policy is, if you aren't on the minor version that has the security release, you aren't being told there is a security release.